Jropp/jason-ropp-js-iobp
# Onboarding Project: Validate name

## What It Does

Properly formats user name entry.

Continues the work on validating information from issue #55 

I added a `titleCaseName` function to the database api that ensures proper phone format on entry and display.

Closes #55 

## How To Test

Enter a name with variations of caps and lower case. It should display properly as 'Last, First' in user list when you hit save. 

## Notes/Caveats

Does not account for any names that need caps outside of the first letter. My apologies to the Irish.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies

- [ ] #56 
- [ ] #58
